aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js

```diff
@@ -92,87 +88,47 @@ const filtersConfig = [
       valueRange: '0 - 40 hours',
```

Comment:
I don't know that `valueRange` applies anymore since you're using Data. Can we remove?